libs/platform/src/lib/components/form/combobox/combobox/combobox.component.ts

```diff
@@ -130,22 +117,21 @@ export class ComboboxComponent extends BaseCombobox implements OnInit, AfterView
     }
 
     onBlur(event: FocusEvent): void {
-        if (this.selected && this.selected.label === this.inputText) {
-            return;
+        const target = event.relatedTarget as HTMLElement;
+        if (target) {
+            const isList = target.closest('.fdp-combobox__list-container');
```

Comment:
`Element.closest()` is not supported in IE11.
We have a function for this `libs/core/src/lib/utils/functions/closest-element.ts`.

Reply:
fixed